zoomus/components/user.py

```diff
@@ -48,7 +48,7 @@ def create(self, **kwargs):
 
     def update(self, **kwargs):
         util.require_keys(kwargs, "id")
-        return self.patch_request("/users/{}".format(kwargs.get("id")), params=kwargs)
+        return self.patch_request("/users/{}".format(kwargs.get("id")), data=kwargs)
```

Comment:
Are you finding this is working for you?  I'm finding for the `/meetings/{}` endpoint I need:

```python
        return self.patch_request(
            "/meetings/{}".format(kwargs.get("id")), params=kwargs, data=kwargs["data"]
        )
```
...where the `data` parameter is explicitly set out of the `kwargs` dictionary.

Reply:
yes the data=kwargs do not work for me. 
I used 
```
 def update(self, **kwargs):
        util.require_keys(kwargs, "id")
        id = kwargs.get("id")
        del kwargs["id"]
        return self.patch_request("/users/{}".format(id), **kwargs)
```

Reply:
Yes, it's working for me, but I don't know if it's the best solution. You can find a working example here: https://github.com/actmd/zoomus/issues/61#issuecomment-606466338